main.go

```diff
@@ -8,11 +8,10 @@ import (
 	"github.com/bndw/pick/config"
 )
 
-// Injected in build
-var version string
+const Version = "v0.5.0"
```

Comment:
Do we need to export the `Version` const? I'd prefer to keep it private and move the declaration into `version.go`.